sidecar/src/service/sidecar_server.rs

```diff
@@ -901,9 +901,10 @@ impl SidecarInterface for SidecarServer {
         session.modify_trace_config(|cfg| {
             update_cfg(cfg.endpoint.take(), |e| cfg.set_endpoint(e), &token);
         });
-        session.configure_dogstatsd(|cfg| {
-            update_cfg(cfg.endpoint.take(), |e| cfg.set_endpoint(e), &token);
-        });
+        // TODO(APMSP-1377): the dogstatsd-client doesn't support test_session tokens yet
+        // session.configure_dogstatsd(|cfg| {
+        //     update_cfg(cfg.endpoint.take(), |e| cfg.set_endpoint(e), &token);
+        // });
```

Comment:
Do you plan to fix this before merging? 
Otherwise, it will prevent dd-trace-php from upgrading, as it will break our tests.

Reply:
I commented this out because it was always broken and as part of migrating I didn't want to _also_ add new functionality. Maybe I misunderstood how it works though so is there a way I can test those tests with this branch before merging?

Reply:
What do you mean by "it was always broken"? :thinking: 

But anyway, you're right it seems unused.
I thought it was used to configure the test_session_token that should be added as a tag to the metric, but actually it's done [on dd-trace-php's side](https://github.com/DataDog/dd-trace-php/pull/2802/files#diff-643c589592ade49e8f68a3c034cfddb87ee2fb6a79452207062e173e3634b8e2R205-R207).